components/06-examples/samenwerken.hbs

```diff
@@ -0,0 +1,55 @@
+{{> '@header' }}
+
+<main>
+  <div class="row">
+    <div class="col-xs-12">
+      <h1>Samenwerken aan behandelen</h1>
+      <h2 class="dso-steps-indicator">
+        Overzicht samenwerkingen
+            </h2>
+        
+    </div>
+      <div class="col-md-3">{{> '@button' label='samenwerking toevoegen' type='button' modifier='link' icon='icon fas fa-plus'}}
+          </div>
+      
+      <div class="col-md-12">
+        <table class="table">
+  <caption class="sr-only">Table met alle samenwerkingen</caption>
+  <thead>
+    <tr>
+     
+      <th scope="col">Naam samenwerking</th>
+      <th scope="col">Status</th>
+      <th scope="col">Initiator</th>
+      <th scope="col">Verzoeknummer</th>
+        <th scope="col">Vertrouwelijkheid</th>
+        <th></th>
+        <th></th>
+        <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+     
+      <td><a href="#fabien" ><b>Aanleg rondweg Leiden</b></a></td>
+      <td>Open</td>
+      <td>Gemeente Den Haag</td>
+      <td>123456789123</td>
+      <td>Geheim</td>
+      <td><a><span class="icon fas fa-users"></span></a></td>
+      <td><a><span class="icon fas fa-pencil-alt"></span></a></td>
+      <td>{{> '@button' label='Openen' type='button' modifier='link pull-right' icon='icon fas fa-angle-right'}}</td>
+    </tr>
+    
+
+    
+    
+  </tbody>
+</table>
+      </div>
+  </div>
+  
+  
+</main>
+    
+{{render '@footer' }}
```

Comment:
Deze hele file moet formatted worden

Reply:
Deze is uitgevoerd. Zo niet corrigeer me waar nodig.